megamek/src/megamek/common/ASFBay.java

```diff
@@ -73,7 +73,7 @@ public boolean canLoad(Entity unit) {
 
         // Only ASFs or Fighter-Mode LAMs
         // (See IO Battleforce section for the rules that allow converted QVs and LAMs to use other bay types)
-        if ((unit.isFighter() && !(unit instanceof FighterSquadron)) || (unit instanceof LandAirMech && unit.getConversionMode() == LandAirMech.CONV_MODE_FIGHTER)) {
+        if (((unit.isFighter()) && (!(unit instanceof FighterSquadron))) || ((unit instanceof LandAirMech) && (unit.getConversionMode() == LandAirMech.CONV_MODE_FIGHTER))) {
```

Comment:
The formatting still needs to be updated here.  This is what you should have:
`if ((unit.isFighter() && !(unit instanceof FighterSquadron)) || ((unit instanceof LandAirMech) && (unit.getConversionMode() == LandAirMech.CONV_MODE_FIGHTER)))`

You only need parents are binary boolean operations, so you don't need parents around the `unit.isFighter()` call.  Also, you don't need the extra parens around the `FighterSquadron` part.

Reply:
Done, though I'm not quite sure I understand the difference for future reference.  What parts of this are the binary Boolean operations?  Just the whole statements on either side of ||  ?

Reply:
binary refers to the fact that there's two operands.  So, something like `e instanceof Aero`, is a binary boolean operation because it has two operands (e, Aero) and returns a boolean (true/false).  I could have been more clear on this in the beginning.

Maybe that's too strict as well, generally we don't require you to paren ands (&&) and ors (||), unless it's necessary for order of operations.  Mostly, it's going to be comparisons (<, >, !=, etc.) and `instanceof` that should have extra parens added.

Reply:
Ok, that makes more sense. I was thinking that Boolean operations are binary by definition.  Did I miss a document that lists all the coding standards, or is it just a learn-as-you-go process?

Reply:
There's no document yet.  There should be.  It's mostly word-of-mouth learn-as-you-go.